frontend/app/common/api.ts

```diff
@@ -157,54 +129,46 @@ export const blockUser = (
   block: boolean;
   site_id: string;
   user_id: string;
-}> => fetcher.put(stringifyUrl(`/admin/user/${id}`, { block: 1, ttl: ttl === 'permanently' ? ttl : undefined }));
+}> => fetcher.put(`/admin/user/${id}`, { block: 1, ttl: ttl === 'permanently' ? ttl : undefined });
 
 export const unblockUser = (
   id: User['id']
 ): Promise<{
   block: boolean;
   site_id: string;
   user_id: string;
-}> => fetcher.put(stringifyUrl(`/admin/user/${id}`, { block: 0 }));
+}> => fetcher.put(`/admin/user/${id}`, { block: 0 });
 
 export const getBlocked = (): Promise<BlockedUser[] | null> => fetcher.get('/admin/blocked');
 
-export const disableComments = (): Promise<void> => fetcher.put(stringifyUrl('/admin/readonly', { url, ro: 1 }));
+export const disableComments = (): Promise<void> => fetcher.put('/admin/readonly', { url, ro: 1 });
 
-export const enableComments = (): Promise<void> => fetcher.put(stringifyUrl('/admin/readonly', { url, ro: 0 }));
+export const enableComments = (): Promise<void> => fetcher.put('/admin/readonly', { url, ro: 0 });
 
 export const uploadImage = (image: File): Promise<Image> => {
   const data = new FormData();
   data.append('file', image);
 
-  return fetcher
-    .post<{ id: string }>('/picture', {
-      headers: {
-        'Content-Type': 'multipart/form-data',
-      },
-      body: data,
-    })
-    .then((resp) => ({
-      name: image.name,
-      size: image.size,
-      type: image.type,
-      url: `${BASE_URL + API_BASE}/picture/${resp.id}`,
-    }));
+  return fetcher.post<{ id: string }>('/picture', {}, data).then((resp) => ({
+    name: image.name,
+    size: image.size,
+    type: image.type,
+    url: `${BASE_URL + API_BASE}/picture/${resp.id}`,
```

Comment:
Seems to be the only use of BASE_URL and API_BASE here, doesn't seem right as well. Should we make a helper function `constructPictureURL` in fetcher instead maybe?